### PR TITLE
added 2 translation lines for .po

### PR DESCRIPTION
**Pull request by dutchy1001**
Lets see if I done it correct  :smiley:

### CODE REVIEW

**Comment by rlf** [comment]
Nope. Not quite.
Comments on the lines

File: uSkyBlock-Core/src/main/java/us/talabrek/ultimateskyblock/event/PlayerEvents.java [hunk 1]
```diff
@@ -226,7 +226,7 @@ private void cancelMemberDamage(Player p1, Player p2, EntityDamageByEntityEvent
         IslandInfo is1 = plugin.getIslandInfo(p1);
         IslandInfo is2 = plugin.getIslandInfo(p2);
         if (is1 != null && is2 != null && is1.getName().equals(is2.getName())) {
-            plugin.notifyPlayer(p1, "\u00a7eYou cannot hurt island-members.");
+            plugin.notifyPlayer (I18nUtil.tr(p1, "\u00a7eYou cannot hurt island-members."));
```
Comment (rlf) [inline uSkyBlock-Core/src/main/java/us/talabrek/ultimateskyblock/event/PlayerEvents.java]: The correct line is:
plugin.notifyPlayer(p1, I18nUtil.tr("\u00a7eYou cannot hurt island-members."));

File: uSkyBlock-Core/src/main/java/us/talabrek/ultimateskyblock/menu/SkyBlockMenu.java [hunk 2]
```diff
@@ -482,7 +482,7 @@ private void addMainMenu(Inventory menu, Player player) {
         menuItem = new ItemStack(Material.SAPLING, 1, (short) 3);
         meta4 = menuItem.getItemMeta();
         meta4.setDisplayName("\u00a7a\u00a7l" + tr("Change Island Biome"));
-        lores.add("\u00a7eCurrent Biome: \u00a7b" + islandInfo.getBiome());
+        lores.add (tr("\u00a7eCurrent Biome: \u00a7b" + islandInfo.getBiome()));
```
Comment (rlf) [inline uSkyBlock-Core/src/main/java/us/talabrek/ultimateskyblock/menu/SkyBlockMenu.java]: The correct line would be:
lores.add(tr("\u00a7eCurrent Biome: \u00a7b{0}", islandInfo.getBiome()));

**Comment by rlf** [comment]
Also, I'd assume some changes to the keys.pot file (and possibly the nl.po).

**Comment by csorrows** [comment]
@dutchy1001 finally getting his coding feet wet?

**Pull request closed without merging**